### PR TITLE
cloud: update docs for serverless billing

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: glossary.md [hunk 1]
```diff
@@ -126,6 +126,11 @@ Regions are generated as data writes increase. The process of splitting is calle
 
 The mechanism of Region split is to use one initial Region to cover the entire key space, and generate new Regions through splitting existing ones every time the size of the Region or the number of keys has reached a threshold.
 
+### Request Unit
+
+The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. It is a metric that allows you to estimate the computational resources required to
+process a specific request in the database. The request unit is also the billing unit for TiDB Cloud Serverless service.
```
Comment (qiancai) [inline glossary.md:132]:
```suggestion
The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. It is a metric that allows you to estimate the computational resources required to process a specific request in the database. The request unit is also the billing unit for TiDB Cloud Serverless service.
```

File: glossary.md [hunk 2]
```diff
@@ -126,6 +126,10 @@ Regions are generated as data writes increase. The process of splitting is calle
 
 The mechanism of Region split is to use one initial Region to cover the entire key space, and generate new Regions through splitting existing ones every time the size of the Region or the number of keys has reached a threshold.
 
+### Request Unit
+
+The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. It is a metric that allows you to estimate the computational resources required to process a specific request in the database. The request unit is also the billing unit for TiDB Cloud Serverless service.
```
Comment (hfxsd) [inline glossary.md:131]: Maybe we can add a link.
```suggestion
The [Request Unit (RU)](https://docs.pingcap.com/tidb/dev/tidb-resource-control#what-is-request-unit-ru) is a unit of measurement used to track the resource consumption of a query or transaction. It is a metric that allows you to estimate the computational resources required to process a specific request in the database. The request unit is also the billing unit for TiDB Cloud Serverless service.
```

File: tidb-cloud/manage-serverless-spend-limit.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+> **Note:**
+>
+> The spend limit is only applicable to Serverless Tier clusters. For Dedicated Tier clusters, TiDB Cloud charges according to the resources that you consume.
+
+For each Serverless Tier cluster, TiDB Cloud provides a free usage quota as follows:
+
+- Row storage: 5 GiB
+- Column storage: 5 GiB
+- Request Units (RU): 50 MB per month
```
Comment (hfxsd) [inline tidb-cloud/manage-serverless-spend-limit.md:16]: Not sure MB is the correct unit for RU. See https://docs.pingcap.com/tidb/dev/tidb-resource-control#what-is-request-unit-ru

File: tidb-cloud/serverless-tier-faqs.md [hunk 4]
```diff
@@ -21,29 +21,27 @@ Get started with the 5-minute [TiDB Cloud Quick Start](/tidb-cloud/tidb-cloud-qu
...
-    - The total size of a single transaction is set to no more than 10 MB on Serverless Tier during the beta phase.
-    - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. In the coming releases, the limitations of the free Serverless Tier might be changed.
-- Some of TiDB Cloud features are partially supported or not supported on Serverless Tier. See [Serverless Tier Limitations](/tidb-cloud/serverless-tier-limitations.md) for details.
+Some of TiDB Cloud features are partially supported or not supported on Serverless Tier. See [Serverless Tier Limitations](/tidb-cloud/serverless-tier-limitations.md) for details.
 
 ### What can Serverless Tier be used for?
 
 You can use your Serverless Tier cluster for non-production workloads such as prototype applications, development environments, hackathons, and academic courses, or to provide temporary data service for your datasets.
 
 ### I created a Developer Tier cluster before Serverless Tier was available. Can I still use my cluster?
```
Comment (hfxsd) [inline tidb-cloud/serverless-tier-faqs.md:42]: Should Developer Tier be Dedicated Tier?

Comment (qiancai) [reply tidb-cloud/serverless-tier-faqs.md:42]: 这里的 Developer Tier 是对的，是给之前还没从  Developer Tier 迁移到 Serverless Tier 集群的用户看的

File: tidb-cloud/manage-serverless-spend-limit.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+title: Manage Spend Limit for Serverless Tier clusters
+summary: Learn how to manage spend limit for your Serverless Tier clusters.
+---
+
+# Manage Spend Limit for Serverless Tier Clusters
+
+> **Note:**
+>
+> The spend limit is only applicable to Serverless Tier clusters. For Dedicated Tier clusters, TiDB Cloud charges according to the resources that you consume.
```
Comment (qiancai) [inline tidb-cloud/manage-serverless-spend-limit.md:10]:
```suggestion
> The spend limit is only applicable to Serverless Tier clusters.
```

File: tidb-cloud/serverless-tier-faqs.md [hunk 6]
```diff
@@ -21,29 +21,27 @@ Get started with the 5-minute [TiDB Cloud Quick Start](/tidb-cloud/tidb-cloud-qu
 
 ### Is Serverless Tier free during beta?
 
-Yes. Serverless Tier is free to use during the beta phase. In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan.
+Until May 30th, 2023, Serverless Tier clusters are still free, with a 0% discount on usage. Starting from June 1st, 2023, usage beyond the free quota will be charged. Once the free quota limit of a cluster is reached, access to the cluster will be restricted until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters).
```
Comment (qiancai) [inline tidb-cloud/serverless-tier-faqs.md:24]:
```suggestion
Until May 30th, 2023, Serverless Tier clusters are still free, with a 0% discount on usage. Starting from June 1st, 2023, usage beyond the free quota will be charged. 
Once the free quota of a cluster is reached, the read and write operation on this cluster will be throttled until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters) or the current usage is reset after the next month.
```

File: tidb-cloud/serverless-tier-limitations.md [hunk 7]
```diff
@@ -1,58 +1,72 @@
...
+
+For each Serverless Tier cluster, TiDB Cloud provides a free usage quota as follows:
+
+- Row storage: 5 GiB
+- Column storage: 5 GiB
+- [Request Units](/tidb-cloud/tidb-cloud-glossary.md#request-unit) (RUs): 50 million RUs per month
+
+The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. It is a metric that allows you to estimate the computational resources required to process a specific request in the database. The request unit is also the billing unit for TiDB Cloud Serverless service.
+
+Once the free quota of a cluster is reached, the read and write operation on this cluster will be throttled until you get more usage quota or the current usage is reset after the next month.
```
Comment (qiancai) [inline tidb-cloud/serverless-tier-limitations.md:66]:
```suggestion
Once the free quota of a cluster is reached, the read and write operation on this cluster will be throttled until you get more usage quota or the current usage is reset after the next month.
```

File: tidb-cloud/manage-serverless-spend-limit.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+For each organization in TiDB Cloud, you can create a maximum of five Serverless Tier clusters by default. To create more Serverless Tier clusters, you need to add a credit card and set a spend limit for the usage. But if you delete some of your previous clusters before creating more, the new cluster can still be created without a credit card.
+
+For each Serverless Tier cluster, TiDB Cloud provides a free usage quota as follows:
+
+- Row storage: 5 GiB
+- Column storage: 5 GiB
+- [Request Units](/tidb-cloud/tidb-cloud-glossary.md#request-unit) (RUs): 50 million RUs per month
+
+Once the free quota of a cluster is reached, the read and write operation on this cluster will be throttled until you get more usage quota or the current usage is reset after the next month.
```
Comment (qiancai) [inline tidb-cloud/manage-serverless-spend-limit.md:22]:
```suggestion
Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you get more usage quota or the current usage is reset after the next month.
```

File: tidb-cloud/select-cluster-tier.md [hunk 9]
```diff
@@ -17,7 +17,21 @@ TiDB Cloud provides the following two options of cluster tiers. Before creating
...
+
+For each Serverless Tier cluster, TiDB Cloud provides a free usage quota as follows:
+
+- Row storage: 5 GiB
+- Column storage: 5 GiB
+- [Request Units](/tidb-cloud/tidb-cloud-glossary.md#request-unit) (RUs): 50 million RUs per month
+
+A Request Unit (RU) is a unit of measure used to represent the amount of resources consumed by a single request to the database. The amount of RUs consumed by a request depends on various factors, such as the operation type or the amount of data being retrieved or modified.
+
+Once the free quota of a cluster is reached, the read and write operation on this cluster will be throttled until you get more usage quota or the current usage is reset after the next month.
```
Comment (qiancai) [inline tidb-cloud/select-cluster-tier.md:32]:
```suggestion
Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you get more usage quota or the current usage is reset after the next month.
```

File: tidb-cloud/serverless-tier-faqs.md [hunk 10]
```diff
@@ -21,29 +21,27 @@ Get started with the 5-minute [TiDB Cloud Quick Start](/tidb-cloud/tidb-cloud-qu
 
 ### Is Serverless Tier free during beta?
 
-Yes. Serverless Tier is free to use during the beta phase. In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan.
+Until May 30th, 2023, Serverless Tier clusters are still free, with a 0% discount on usage. Starting from June 1st, 2023, usage beyond the free quota will be charged. Once the free quota of a cluster is reached, the read and write operation on this cluster will be throttled until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters) or the current usage is reset after the next month.
```
Comment (qiancai) [inline tidb-cloud/serverless-tier-faqs.md:24]:
```suggestion
Until May 30th, 2023, Serverless Tier clusters are still free, with a 0% discount on usage. Starting from June 1st, 2023, usage beyond the free quota will be charged. Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters) or the current usage is reset after the next month.
```

File: tidb-cloud/serverless-tier-limitations.md (hunk 7 above)
Comment (qiancai) [inline tidb-cloud/serverless-tier-limitations.md:66]:
```suggestion
Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you get more usage quota or the current usage is reset after the next month.
```

File: tidb-cloud/manage-serverless-spend-limit.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+- Row storage: 5 GiB
+- Column storage: 5 GiB
+- [Request Units](/tidb-cloud/tidb-cloud-glossary.md#request-unit) (RUs): 50 million RUs per month
+
+Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you get more usage quota or the current usage is reset after the next month.
+
+To learn more about the RU consumption of different resources (including read, write, SQL CPU, and network egress), the pricing details, and the throttled information, see [TiDB Cloud Serverless Tier Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details).
+
+If you want to create a Serverless Tier cluster with an additional quota, you can edit the spend limit on the cluster creation page. For more information, see [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md#step-4-create-a-tidb-cluster).
+
```
Comment (hfxsd) [inline tidb-cloud/manage-serverless-spend-limit.md:27]:
```suggestion

## Update spend limit

```

File: tidb-cloud/manage-serverless-spend-limit.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+# Manage Spend Limit for Serverless Tier Clusters
+
+> **Note:**
+>
+> The spend limit is only applicable to Serverless Tier clusters.
+
+Spend limit refers to the maximum amount of money that you are willing to spend on a particular workload in a month. It is a cost-control mechanism that allows you to set a budget for your Serverless Tier clusters.
+
+For each organization in TiDB Cloud, you can create a maximum of five Serverless Tier clusters by default. To create more Serverless Tier clusters, you need to add a credit card and set a spend limit for the usage. But if you delete some of your previous clusters before creating more, the new cluster can still be created without a credit card.
+
```
Comment (hfxsd) [inline tidb-cloud/manage-serverless-spend-limit.md:15]:
```suggestion

## Free usage quota

```

File: tidb-cloud/serverless-tier-limitations.md [hunk 13]
```diff
@@ -1,58 +1,72 @@
...
 
 - [Statement Analysis](/tidb-cloud/tune-performance.md#statement-analysis) and [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer) are unavailable for Serverless Tier.
 
-## Stream data
+### Stream data
 
 * [Changefeed](/tidb-cloud/changefeed-overview.md) is not supported for Serverless Tier currently.
 * [Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md) is not supported for Serverless Tier currently.
+
+## Usage quotas
```
Comment (hfxsd) [inline tidb-cloud/serverless-tier-limitations.md:54]:
```suggestion
## Usage quota
```

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/serverless-tier-limitations.md [hunk 14]
```diff
@@ -1,58 +1,71 @@
...
+
+Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you get more usage quota or the current usage is reset after the next month. For example, once the storage exceeds 5 GiB, the maximum size limit of a single transaction is reduced from 10 MiB to 1 MiB.
+
+To learn more about the RU consumption of different resources (including read, write, SQL CPU, and network egress), the pricing details, and the throttled information, see [TiDB Cloud Serverless Tier Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details).
+
+If you want to create a Serverless Tier cluster with an additional quota, you can edit the spend limit on the cluster creation page. For more information, see [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md#step-4-create-a-tidb-cluster).
+
+After creating a Serverless Tier, you can still check and edit the spend limit on your cluster overview page. For more information, see [Manage Spend Limit for Serverless Tier Clusters](manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters).
```
Comment (qiancai) [inline tidb-cloud/serverless-tier-limitations.md:71]:
```suggestion
After creating a Serverless Tier, you can still check and edit the spend limit on your cluster overview page. For more information, see [Manage Spend Limit for Serverless Tier Clusters](/tidb-cloud/manage-serverless-spend-limit.md).
```

File: tidb-cloud/serverless-tier-faqs.md [hunk 15]
```diff
@@ -21,29 +21,27 @@ Get started with the 5-minute [TiDB Cloud Quick Start](/tidb-cloud/tidb-cloud-qu
 
 ### Is Serverless Tier free during beta?
 
-Yes. Serverless Tier is free to use during the beta phase. In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan.
+Until May 30, 2023, Serverless Tier clusters are still free, with a 100% discount on usage. After that, usage beyond the free quota will be charged. Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters) or the current usage is reset after the next month.
```
Comment (qiancai) [inline tidb-cloud/serverless-tier-faqs.md:24]:
```suggestion
Until May 30, 2023, Serverless Tier clusters are still free, with a 100% discount on usage. After that, usage beyond the free quota will be charged. Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md) or the current usage is reset after the next month.
```

**Comment by baiyuqing** [comment]
LGTM

File: tidb-cloud/manage-serverless-spend-limit.md [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+## Usage quota
+
+For each Serverless Tier cluster, TiDB Cloud provides a free usage quota as follows:
```
Comment (baiyuqing) [inline tidb-cloud/manage-serverless-spend-limit.md:18]: Only maximum five of serverless clusters have free usage quota.

File: tidb-cloud/select-cluster-tier.md [hunk 17]
```diff
@@ -17,7 +17,20 @@ TiDB Cloud provides the following two options of cluster tiers. Before creating
 
 The TiDB Cloud Serverless Tier (previously called Developer Tier) is a fully managed service of TiDB. It's still in beta and should not be used in production. However, you can use Serverless Tier clusters for non-production workloads such as prototype applications, hackathons, academic courses, or to provide a temporary data service for your datasets.
 
-For each TiDB Cloud account, you can create a maximum of five complimentary Serverless Tier clusters to use during the beta phase.
+### Usage quota
+
+For each organization in TiDB Cloud, you can create a maximum of five Serverless Tier clusters by default. To create more Serverless Tier clusters, you need to add a credit card and set a [spend limit](/tidb-cloud/tidb-cloud-glossary.md#spend-limit) for the usage.
+
+For each Serverless Tier cluster, TiDB Cloud provides a free usage quota as follows:
```
Comment (baiyuqing) [inline tidb-cloud/select-cluster-tier.md:24]: Only maximum of five complimentary Serverless Tier clusters  have free usage quota.

**Review by shhdgit — APPROVED** [review]

**Comment by baiyuqing** [comment]
LGTM

**Review by baiyuqing — APPROVED** [review]

File: tidb-cloud/manage-serverless-spend-limit.md (hunk 16 above)
Comment (qiancai) [reply tidb-cloud/manage-serverless-spend-limit.md]: updated in https://github.com/pingcap/docs/pull/13335/commits/19d6207a48bdc66b6aeb6d280fa9bad773aa2ce0

**Pull request merged**